### DISCUSSION TITLE
Shapiro Test Error

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#274] [Q&A] [answered]
Halo saya mengalami kendala error saat ingin menjalankan shapiro test.
![Error1](https://user-images.githubusercontent.com/107049922/185791705-b3ca18a5-599b-4bf9-992e-66201c676ae5.png)
Ini saya menggunakan model step-wise backwards dengan data dari https://www.kaggle.com/datasets/greenwing1985/housepricing.

Mohon bantuannya ya, terima kasih!

**Comment by anugrah18003** [comment, marked as answer]
Dear @RanggaGemilang 

Pada uji normalitas menggunakan Shapiro-Wilk normality test memiliki limitasi data yang digunakan, dimana pada uji Shapiro sample data harus berada pada rentang 3 hingga 5000. Sedangkan data yang digunakan tidak ada pada rentang tersebut sehingga terjadi error.

Sebagai alternatif, jika sample data yang digunakan lebih dari 5000 bisa menggunakan Kolmogorov-Smirnov test. Dalam R, untuk melakukan uji normalitas kolmogorov-smirnov dapat menggunakan fungsi **ks.test()**.
Dengan kriteria uji: p-value > α = 0.05, maka data berdistribusi normal.

Berikut juga saya lampirkan referensi untuk metode Uji normalitas di R : https://www.statology.org/test-for-normality-in-r/
